frontend/views/pages/GroupChat.vue

```diff
@@ -16,37 +16,56 @@ page(pageTestName='dashboard' pageTestHeaderName='groupName')
             i18n Channel Options
 
           ul
-            menu-item(@click='openModal("EditChannelNameModal")')
+            menu-item(
+              v-if='!summary.general && ourUsername === summary.creator'
+              @click='openModal("EditChannelNameModal")'
+              data-test='renameChannel'
+            )
               i18n Rename
-            menu-item(@click='openModal("GroupMembersAllModal", {name: summary.title})')
+            menu-item(@click='openModal("ChatMembersAllModal")')
               i18n Members
-            menu-item.c-separator(@click='openModal("UserSettingsModal", {section: "notifications"})')
+            menu-item(
+              :class='`${!summary.general ? "c-separator" : ""}`'
+              @click='openModal("UserSettingsModal", {section: "notifications"})'
+              data-test='notificationsSettings'
+            )
               i18n Notifications settings
-            menu-item(@click='openModal("LeaveChannelModal")')
-              i18n(:args='{ groupName: summary.title }') Leave {groupName}
-            menu-item.has-text-danger(@click='openModal("DeleteChannelModal")')
+            menu-item(
+              v-if='!summary.general'
+              @click='openModal("LeaveChannelModal")'
+              data-test='leaveChannel'
+            )
+              i18n(:args='{ channelName: summary.title }') Leave {channelName}
+            menu-item.has-text-danger(
+              v-if='!summary.general && ourUsername === summary.creator'
+              @click='openModal("DeleteChannelModal")'
+              data-test='deleteChannel'
+            )
               i18n Delete channel
 
   template(#description='')
     .c-header-description
       i18n.is-unstyled.c-link(
         tag='button'
-        @click='openModal("GroupMembersAllModal")'
+        @click='openModal("ChatMembersAllModal")'
         :args='{ numMembers: members.size  }'
+        data-test='channelMembers'
       ) {numMembers} members
       | ∙
-      .is-unstyled.c-link(
-        tag='button'
+      .is-unstyled(
+        :class='{"c-link": ourUsername === summary.creator}'
```

Comment:
The ∙ has difference spacing before the channel description depending on whether or not we're the channel creator:

<img width="280" alt="Screen Shot 2022-03-22 at 8 35 32 PM" src="https://user-images.githubusercontent.com/138706/159618897-f9eaeda2-797e-4837-a8f1-76fb4c0a3f3b.png">

vs

<img width="293" alt="Screen Shot 2022-03-22 at 8 35 29 PM" src="https://user-images.githubusercontent.com/138706/159618923-8265f106-36b9-4b5e-b947-2016d2c817c5.png">

Reply:
Oh. Very much sensitive. 👍